code/game/atoms_movable.dm

```diff
@@ -267,6 +267,10 @@
 		src.thrower = null
 		src.throw_source = null
 
+	if(ishuman(thrower)) //Human specific procs
+		var/mob/living/carbon/human/H = thrower
```

Comment:
Why bother casting this to human if you defined the proc as mob/living

Reply:
Use redefined procs instead of type checks whenever possible.